tox.ini

```diff
@@ -6,8 +6,10 @@ envlist =
 isolated_build = true
 
 [testenv]
+setenv =
+    PYVISTA_OFF_SCREEN = True
```

Comment:
What happens if you run the tests locally now?

Reply:
It works fine after adding xvfb, locally and on the azure platform as well